styles/github/catppuccin.user.css

```diff
@@ -386,87 +384,145 @@
+      #repos-file-tree button[aria-label="Search this repository"],
+      #repos-file-tree span[aria-label="Add file"] a,
       .gtXnmh .Box-sc-g0xbh4-0.jGfYmh > * {
-        background-color: @surface0;
-        border-color: @surface1;
-        &:hover {
-          background-color: @surface1;
-          border-color: @surface2;
-        }
-        & * {
-          fill: @text;
-          color: @text;
-        }
-      }
-      .cPEOjV {
-        background-color: @mantle;
-      }
-      .Box-sc-g0xbh4-0.bUCzHg,
-      .ehcSsh > * {
-        border-color: @surface1 !important;
+      background-color: @surface0;
+      border-color: @surface1;
+      &:hover {
+        background-color: @surface1;
+        border-color: @surface2;
       }
-      .eYedVD .Box-sc-g0xbh4-0.jGfYmh *,
-      .Text-sc-17v1xeu-0.foaUgQ {
-        color: @subtext0;
-      }
-      .hRMLtq,
-      .ilcYjX *,
-      .lcModf,
-      button.types__StyledButton-sc-ws60qy-0.kRuNkv {
+      & * {
+        fill: @text;
         color: @text;
       }
-      .Flash__StyledFlash-sc-hzrzfc-0.kUDiXM {
-        background-color: fadeout(@peach, 90%);
-        border-color: @peach;
-      }
-      .kUDiXM svg {
-        fill: @peach;
-      }
-      /* Some popup */
-      .iGzDhH,
-      .hBobmI,
-      #__primerPortalRoot__ > div > div > div {
-        background-color: @mantle;
-      }
-      .Text-sc-17v1xeu-0.gPDEWA {
-        color: @subtext1;
-      }
-      .TabNav__TabNavTabList-sc-pwdi4r-1.gFzOft {
-        & * {
-          border-color: @surface0;
+    }
+    .cPEOjV,
+    .gtXnmh {
+      background-color: @mantle;
+    }
+    /* GitHub decided to manually override these links' colors for whatever reason */
+    .GCHqa,
+    .QBrkF {
+      color: @accent-color;
+    }
+    .QBrkF {
+      background-color: fade(@accent-color, 10%);
+    }
+    .Box-sc-g0xbh4-0.bUCzHg,
+    .ehcSsh > * {
+      border-color: @surface1 !important;
+    }
+    .eYedVD .Box-sc-g0xbh4-0.jGfYmh *,
+    .Text-sc-17v1xeu-0.foaUgQ {
+      color: @subtext0;
+    }
+    .hRMLtq,
+    .ilcYjX *,
+    .lcModf,
+    button.types__StyledButton-sc-ws60qy-0.kRuNkv {
+      color: @text;
+    }
+    .Flash__StyledFlash-sc-hzrzfc-0.kUDiXM {
+      background-color: fadeout(@peach, 90%);
+      border-color: @peach;
+    }
+    .kUDiXM svg {
+      fill: @peach;
+    }
+    /* Some popup */
+    .iGzDhH,
+    .hBobmI,
+    #__primerPortalRoot__ > div > div > div {
+      background-color: @mantle;
+      [class^="TabNav__TabNavTabList-"] button {
+        &.selected {
+          background-color: @mantle;
+          border-color: @surface1;
+        }
+        &:not(.selected) {
+          color: @overlay2;
         }
         &:hover {
-          & * {
-            background: @mantle;
-          }
+          background-color: @surface1;
         }
       }
+      .eJLRZa {
+        border-color: @surface1;
+      }
+    }
+    .Text-sc-17v1xeu-0.gPDEWA {
+      color: @subtext1;
+    }
```

Comment:
Next order of business is trying to figure out what @isabelroses was styling here and add comments explaining it... send help.

Reply:
its for this 
![image](https://github.com/catppuccin/userstyles/assets/71222764/fb675719-733a-408d-b443-eeaf6c788c5e)